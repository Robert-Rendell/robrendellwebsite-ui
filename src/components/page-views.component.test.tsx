import React from "react";
import { render } from "@testing-library/react";
import { PageViewsComponent } from "./page-views.component";

describe("PageViewsComponent", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<PageViewsComponent />);
    expect(asFragment()).toMatchSnapshot();
  });
});
