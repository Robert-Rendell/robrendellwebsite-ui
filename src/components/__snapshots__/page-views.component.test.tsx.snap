// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageViewsComponent should match snapshot 1`] = `
<DocumentFragment>
  Minds unlocked: 
  <img
    height="50"
    src="infinity-spinner.svg"
  />
</DocumentFragment>
`;
